l10n_ro_config/models/res_company.py

```diff
@@ -147,7 +147,7 @@ def _check_is_l10n_ro_record(self, company=False):
             company = self
         else:
             company = self.browse(company)
-        return company.account_fiscal_country_id.code == "RO"
+        return company.l10n_ro_accounting
```

Comment:
Mai este necesar campul  l10n_ro_accounting? Nu putem sa renuntam la el si sa folosim  company.account_fiscal_country_id.code == "RO" ?

Reply:
Nu, pentru ca trebuie sa avem posibilitatea sa activam si dezactivam partea asta, codul pe Romania il folosim si in alte tari, si doar dupa fiscal country nu putem face.